templates/repo/issue/view_content/reactions.tmpl

```diff
@@ -0,0 +1,17 @@
+{{if .Reactions}}
```

Comment:
nit: remove this if statement. There's no reason to pass in a nil `.Reactions`, since you could/should just use the `add_reaction` template instead.